app-backend/tile/src/main/resources/application.conf

```diff
@@ -7,3 +7,36 @@ tile-server {
   bucket = "rasterfoundry-staging-catalogs-us-east-1"
   bucket = ${?TILE_SERVER_BUCKET}
 }
+
+kamon {
+  metric {
+    filters {
+      trace.includes = [ "**" ]
+    }
+  }
+
+  subscriptions {
+    trace                = [ "**" ]
+    trace-segment        = [ "**" ]
+    akka-http-server     = [ "**" ]
+  }
+
+  statsd {
+    hostname = "statsd"
+    port = 8125
+    simple-metric-key-generator.include-hostname = false
```

Comment:
I added this to remove the host portion of the metric. Otherwise, we end up with a new metric every time a new container instance is spun up.